### PR TITLE
Remove install setup and use from common repo

### PR DESCRIPTION
**Pull request by Koncpa**
Also adjust upstream .fmf plan and check channel
just under condition, if it's deployment provided
via oc client.

### CODE REVIEW

File: Setup/creating_test_namespace/runtest.sh [hunk 1]
```diff
@@ -59,6 +59,8 @@ rlJournalStart
         rlRun "ocpopCheckPodStateAndContinues Running ${TIMEOUT_CONTROLLER_KEEPS_RUNNING} ${OPERATOR_NAMESPACE} ${controller_name}" 0 \
               "Checking controller POD continues Running [${TIMEOUT_CONTROLLER_KEEPS_RUNNING} secs.]"
 	#SECENGSP-5573 Issue
-	rlRun "ocpopCheckOperatorChannel tang-operator stable"
+        if [ "${OPERATOR_DEPLOYMENT_CLI}" == "true" ]; then
```
[the remark this answers is not in the document]
Comment (Koncpa) [reply Setup/creating_test_namespace/runtest.sh:62]: It'll be defined for downstream plans where is used deployment via CLI.

[the remark this answers is not in the document]
Comment (sarroutbi) [reply Setup/creating_test_namespace/runtest.sh:62]: > It'll be defined for downstream plans where is used deployment via CLI.

Then, if possible, I would change it to contain something that references DOWNSTREAM, for it to be more clear.

**Comment by Koncpa** [comment]
There is issue with version of GO on used releases is version of GO around 1.21, but when building a image need go >= 1.22.0. 

`go: downloading sigs.k8s.io/controller-runtime/tools/setup-envtest v0.0.0-20240327193027-21368602d84b
go: sigs.k8s.io/controller-runtime/tools/setup-envtest@latest: sigs.k8s.io/controller-runtime/tools/setup-envtest@v0.0.0-20240327193027-21368602d84b requires go >= 1.22.0 (running go 1.21.8; GOTOOLCHAIN=local)
make: *** [Makefile:247: /var/tmp/tang-operator_sources/bin/setup-envtest] Error 1`

What we need to is use rawhide release, where we could get newer version of GO, but there is another issue with panic error of building image which we discussed before. My suggestion is run it just for Fedora-Rawhide and upgrade version for GO 1.22, but we need to handle that panic errors. @sarroutbi WDYT?

**Comment by sarroutbi** [comment]
> There is issue with version of GO on used releases is version of GO around 1.21, but when building a image need go >= 1.22.0.
> 
> `go: downloading sigs.k8s.io/controller-runtime/tools/setup-envtest v0.0.0-20240327193027-21368602d84b go: sigs.k8s.io/controller-runtime/tools/setup-envtest@latest: sigs.k8s.io/controller-runtime/tools/setup-envtest@v0.0.0-20240327193027-21368602d84b requires go >= 1.22.0 (running go 1.21.8; GOTOOLCHAIN=local) make: *** [Makefile:247: /var/tmp/tang-operator_sources/bin/setup-envtest] Error 1`
> 
> What we need to is use rawhide release, where we could get newer version of GO, but there is another issue with panic error of building image which we discussed before. My suggestion is run it just for Fedora-Rawhide and upgrade version for GO 1.22, but we need to handle that panic errors. @sarroutbi WDYT?

Sorry, I don't understand your point ... it seems that the fix works properly in Rawhide, right? I still need to review the [issue with Go 1.22](https://github.com/golang/go/issues/64812), but I don't believe this is the issue here, as Rawhide seems to work appropriately ...

BTW, why do you need to compile the code? Shouldn't it be enough to download "latest" image from quay.io?

**Comment by Koncpa** [comment]
And about compilation, we agreed, that could be useful in case, when we would like to check how upstream changes affect testing..also in future, when we setup CI for upstream code, we could use that upstream task for testing in upstream PR's. It's much easier to compile it, than every change upload it to quay.io..

**Comment by sarroutbi** [comment]
My suggestion to fix this, at this moment, is to not build the operator (this should not be the tests mission), and use, if possible, the image: `quay.io/sec-eng-special/tang-operator-bundle:latest`, as this is the latest official image for tang-operator

**Comment by Koncpa** [comment]
I would not set high priority here, but I would at least file that issue. Everybody, who will try to build upstream could met with same issue.

**Comment by sarroutbi** [comment]
> For now we use both option, but I would at least open issue for it in tang-operator upstream to address issue, with panic errors.

Absolutely. It was opened some time ago: https://github.com/latchset/tang-operator/issues/248

**Comment by sarroutbi** [comment]
> > For now we use both option, but I would at least open issue for it in tang-operator upstream to address issue, with panic errors.
> 
> Absolutely. It was opened some time ago: [latchset/tang-operator#248](https://github.com/latchset/tang-operator/issues/248)
> 
> I have it in my TODO list

@Koncpa : [latchset/tang-operator#248](https://github.com/latchset/tang-operator/issues/248) has been resolved. It should work now

**Review by sarroutbi — APPROVED** [review]
Changes LGTM

**Pull request merged**